Gemfile

```diff
@@ -12,3 +12,4 @@ gem 'tzinfo-data', platforms: [:mswin, :mingw, :jruby]
 gem 'middleman', '>= 4.0.0'
 gem 'middleman-livereload'
 gem "middleman-sprockets"
+gem 'sass'
```

Comment:
Very strange

Reply:
This shouldn't be here. Naughty.

It's because I branched off https://github.com/alphagov/govwifi-product-page/pull/46 , as that's required for `make serve`. I'll remove it from this PR shortly.